src/index.js

```diff
@@ -0,0 +1,11 @@
+if (process.env.NODE_ENV !== "production") {
+    require("dotenv").config();
+  }
```

Comment:
Take care with this kind of changes (blank spaces), please be sure you have these plugins:
Integrated Terminal
Prettier
Material Icon Theme